src/promptflow/promptflow/_sdk/_service/utils/utils.py

```diff
@@ -100,6 +104,19 @@ def make_response_no_content():
     return make_response("", 204)
 
 
+def check_pfs_service_status(pfs_port) -> bool:
+    """Check if pfs service is running."""
+    try:
+        response = requests.get("http://localhost:{}/heartbeat".format(pfs_port))
+        if response.status_code == 200:
+            logger.info(f"Pfs service is already running on port {pfs_port}.")
```

Comment:
not sure on this level, maybe debug is better? @wangchao1230

Reply:
changed to debug level